docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+# Small local network of two nodes using the fastest mean16
+version: '3'
+services:
+  node1:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile-epoch
+    image: aetrnty/epoch
+    hostname: node1
+    environment:
+      EPOCH_CONFIG: /home/epoch/epoch.yaml
+    volumes:
+      - ${PWD}/docker/epoch_node1_mean16.yaml:/home/epoch/epoch.yaml
+      - node1_db:/home/epoch/node/data/mnesia
```

Comment:
I tested that persisted DB in docker-compose works. I have not tested that bind mount in https://github.com/aeternity/epoch/blob/067d8dc15a2f76f3d2b38e087020bfa2bf75a905/docs/docker.md#persisting-data was not broken.

Reply:
It's hard to tell because the image is suitable for UAT only and hard to join the INT with it. As if you build your own, you'll have the `master` branch source code with it.

Reply:
I mean, you have to change the genesis accounts too.